src/services/services.tsx

```diff
@@ -0,0 +1,12 @@
+const backend_URL: string = 'https://lessoner.herokuapp.com/en';
```

Comment:
let's move that file to `src/services/api/isEmailExists.ts`

Reply:
done